pages/portfolio.html

```diff
@@ -1,12 +1,92 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Atefeh Mirzaei, portfolio</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Caveat&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/styles/style.css">
+  </head>
+  <body>
+    <main>
+      <!-- ***********************  Header / Nav-bar  *********************** -->
+      <header>
+        <section class="content-wrap">
+          <h2>Atefeh Mirzaei</h2>
+            <ul class="nav-bar">
```

Comment:
I would recommend using the `<nav>` tag here rather than assigning a nav-bar class to this `<ul>` tag. It's a little more semantic. :)